src/routes/console/project-[project]/auth/user-[user]/updateEmail.svelte

```diff
@@ -36,6 +36,7 @@
 <Form onSubmit={updateEmail}>
     <CardGrid>
         <Heading tag="h6" size="7">Email</Heading>
+        <p>Update user's email. An Email should be formatted as: <b>name@example.com.</b></p>
```

Comment:
```suggestion
        <p>Update user's email. An Email should be formatted as: <b>name@example.com</b>.</p>
```

Do we have any option to set the value as a code example?

Reply:
Do you mean just formatting it like inline code? It's possible with pink, but we don't use it anywhere in the Console.